src/routes.php

```diff
@@ -1,7 +1,13 @@
 <?php
 if (env('ENABLE_DOC_US', false)) {
     Route::get('schema', function (Illuminate\Http\Request $request) {
-        $schema = UniSharp\DocUs\Parser::getSchema();
+        $exclude = [];
+
+        if ($request->exclude !== null) {
+            $exclude = explode(',', strtolower($request->exclude));
```

Comment:
why with `strtolower`?

Reply:
might relate to MySQL `lower_case_table_names` config. 
the users should have the control.

Reply:
Thanks for your reminder. I got it. I forgot to consider this configuration of MySQL.
I have removed `strtolower` on my last commit.